flytectl/cmd/compile/compile.go

```diff
@@ -105,6 +100,65 @@ func compileFromPackage(packagePath string) error {
 	return nil
 }
 
+func handleWorkflow(
+	workflow *admin.WorkflowSpec,
+	compiledTasks []*core.CompiledTask,
+	compiledWorkflows map[string]*core.CompiledWorkflowClosure,
+	compiledLaunchPlanProviders []common.InterfaceProvider,
+	plans map[string]*admin.LaunchPlan,
+	workflows map[string]*admin.WorkflowSpec) ([]common.InterfaceProvider, error) {
+	reqs, _ := compiler.GetRequirements(workflow.Template, workflow.SubWorkflows)
+	wfName := workflow.Template.Id.Name
+
+	// Check if all the subworkflows referenced by launchplan are compiled
+	for i := range reqs.GetRequiredLaunchPlanIds() {
+		lpID := &reqs.GetRequiredLaunchPlanIds()[i]
```

Comment:
Nit: Can we do sth like this?

```go
for i, lpID := range reqs.GetRequiredLaunchPlanIds() {
```

Reply:
There is a mutex on the launchplans & a linter proposed changing to the current form.

Reply:
Ok 👌